coc-settings.json

```diff
@@ -23,12 +25,19 @@
         "dart": {
             "command": "dart",
             "args": [
-                "/usr/lib/dart/bin/snapshots/analysis_server.dart.snapshot",
+                "/usr/local/opt/dart/libexec/bin/snapshots/analysis_server.dart.snapshot",
```

Comment:
This should probably be configured somewhere as it looks it depends on user specific paths

Reply:
yes, I think so 🤔  this kinds of config are very specific maybe we should not added it in here for terraform and dart I mean